### DISCUSSION TITLE
Similar to Obtanium to update local repositories in Windows

### DISCUSSION THREAD

**Discussion by Ocanamat** [ImranR98/Obtainium#1300] [Q&A]
Does anybody know of a project similar to Obtainium to pull new releases of followed, starred, or locally downloaded repositories in Windows? I know there is the Github Desktop client, but it does come a little short when I just want to pull releases and just keep up with the updated projects' repositories as far as the interface and pulling just the binaries goes. 

Any tip or info is kindly appreciated  (-:

**Comment by Ocanamat** [comment]
For future reference, found this on the discussion asking the same thing but for linux:

> Someone had mentioned https://github.com/marcosnils/bin in some issue
> 
> Edit: Mentioned in #777
> 

_Originally posted by @ImranR98 in https://github.com/ImranR98/Obtainium/discussions/1053#discussioncomment-7394610_

It does the job, ui-less but does the job :-)

**Comment by trymeouteh** [comment]
https://github.com/ImranR98/Obtainium/discussions/1760
